skimage/io/collection.py

```diff
@@ -157,7 +157,24 @@ def vidread_step(f, step):
       x = ic[0]  # calls vidread_step(video_file, step=3)
       x[5]  # is the sixth element of a list of length 8 (24 / 3)
 
-    Another use of ``load_func`` would be to convert all images to ``uint8``::
+    Alternatively, if a `load_func` is provided and `load_pattern` is a
+    sequence, an ImageCollection of corresponding length will be created,
+    and the individual images will be loaded by calling `load_func` with the
+    matching element of the `load_pattern` as its first argument. In this
+    case, the elements of the sequence do not need to be resolvable file
```

Comment:
I didn't understand "resolvable" here. Could we use a nice, simpler synonym? :slightly_smiling_face:

Reply:
Hi @alexdesiqueira, sorry for the late response. "resolvable" here means that it is a file name that exists on the local file system. I'll propose a changed wording in the next commit.